IapExample/App.js

```diff
@@ -119,6 +119,10 @@ class Page extends Component {
       if (Platform.OS === 'android') {
         await RNIap.flushFailedPurchasesCachedAsPendingAndroid();
       } else {
+        // WARNING This line should not be included in production code
+        // This call will call finishTransaction in all pending purchases on every launch,
+        // effectively consuming purchases that you might not have verified the receipt or given the consumer their product
+        // TD:DR you will no longer receive any updates from Apple on every launch for pending purchases
```

Comment:
What is TD:DR? I've only heard about TL;DR so sorry if I don't understand this 🙏

Reply:
It's a typo :)